pkgs/by-name/di/difftastic/package.nix

```diff
@@ -5,50 +5,37 @@
   fetchFromGitHub,
   testers,
   difftastic,
+  nix-update-script,
 }:
 
-let
-  mimallocPatch = fetchpatch {
```

Comment:
The `fetchpatch` argument in line 3 can be removed now

Reply:
Oops. Thought I nixed it before.